test/documentation/bio-doxygen-package.cmake

```diff
@@ -0,0 +1,15 @@
+
+cmake_minimum_required (VERSION 3.7)
+
+set (CPACK_GENERATOR "TXZ")
+
+set (CPACK_COMPONENTS_ALL doc)
+
+set (CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+
+set (CPACK_PACKAGE_VENDOR "bio")
+# A description of the project, used in places such as the introduction screen of CPack-generated Windows installers.
+# set (CPACK_PACKAGE_DESCRIPTION_FILE "") # TODO
+set (CPACK_PACKAGE_CHECKSUM "SHA256")
+
+include (CPack)
```

Comment:
What does this do?

Reply:
I have no clue

Reply:
Yeah, I don't know if we ever want to package the documentation separately... but it is a small file so I don't care. This could also be symlinked to the SeqAn3 submodule tough!

Reply:
This file is adapted in line 10 from bio to seqan. I don't if it makes sense to include it from seqan isntead of copying. I can just remove the packaging stuff for now